pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_publishing-a-module-in-pantheon.adoc

```diff
@@ -0,0 +1,36 @@
+include::../attributes.adoc[]
+[id="publishing-a-module-in-pantheon_{context}"]
+= Publishing a module in {ProductShortName}
+
+Publish a module in {ProductShortName} to the customer portal.
+
+.Prerequisites
+
+* You are assigned the publisher role in {ProductShortName}.
+* You are logged-in.
+* The module to be published is uploaded.
+
+.Procedure
+
+. Navigate to link:https://pantheon.cee.redhat.com/#/search[].
+
+. Click the name of the module you want to publish.
++
+If you don't see the module listed, search for the modules using the search box.
+
+. In the list of module versions, click the three verical dots next to the version of the module you want to publish.
+
+. Click *Edit metadata* to add the following publishing details for the module:
```

Comment:
I wonder whether adding metadata should be a separate task - this strikes me as something users might do independently of publishing a module. We don't have an issue for that just yet, though, so perhaps we can remove these steps from here but keep the content until we do - would that work for you?

Reply:
Indeed, I checked. The metadata persists across sessions, so it seems like adding metadata is a separate task. I've commented out the step to retain the content.

Reply:
Sorry to jump in here but I 'd like to mention that, a user cannot Publish a module until metadata is successfully added. We might want to  consider adding that as a task/step before clicking on Publish button step to let the users know about the expected workflow. Let me know what do you think?

Reply:
Thanks @anarang! 

@adahms, I wonder whether we expect that metadata is already updated when a user is reading these instructions; in that case I can have "metadata is added for the module" as a prerequisite. Any thoughts Anisha?

Reply:
Hey @theashiot - thanks for the callout.

I agree - based on Anisha's description, this sounds like a great candidate for a prerequisite. We'll create modules on actually assigning and updating metadata later on, but for now, calling it out as something that needs to happen sounds good to me.

Reply:
Hi @anarang,

I added the following pre-requisite to the module:

"The metadata required for publishing has been applied to the module." 

 Is this ok?

best,
ashwin

Reply:
@theashiot sounds good! thanks. :)